packages/core/admin/ee/server/services/review-workflows/stages.js

```diff
@@ -21,9 +21,8 @@ module.exports = ({ strapi }) => {
       return strapi.entityService.findMany(STAGE_MODEL_UID, params);
     },
 
-    findById(id, { workflowId, populate }) {
+    findById(id, { populate } = {}) {
       const params = {
-        filters: { workflow: workflowId },
```

Comment:
I removed this filter... Do we need it? If you specify the Stage id , filtering by the workflow id should not be necessary.

Reply:
That makes sense to me, all stages even across different workflows should have unique IDs